src/app/[locale]/viewer/_components/submodel/hierarchical-structures/HierarchicalStructuresDetail.tsx

```diff
@@ -33,15 +38,60 @@ export function HierarchicalStructuresDetail(props: HierarchicalStructuresDetail
         return;
     });
 
+    const ArcheTypeSemanticId = 'https://admin-shell.io/idta/HierarchicalStructures/ArcheType/1/0';
```

Comment:
should we put stuff like this also in its own enum like the submodel semantic Ids?

Reply:
yes your're right. I put it in the existing SubmodelElementSemanticId enum.